lua/lspconfig/server_configurations/lua_ls.lua

```diff
@@ -49,26 +49,26 @@ Completion results will include a workspace indexing progress message until the
 
 ```lua
 require'lspconfig'.lua_ls.setup {
-  settings = {
-    Lua = {
-      runtime = {
-        -- Tell the language server which version of Lua you're using (most likely LuaJIT in the case of Neovim)
-        version = 'LuaJIT',
-      },
-      diagnostics = {
-        -- Get the language server to recognize the `vim` global
-        globals = {'vim'},
-      },
-      workspace = {
+  on_init = function(client)
+    local path = client.workspace_folders[1].name
+    if not vim.uv.fs_stat(path..'/.luarc.json') and not vim.uv.fs_stat(path..'/.luarc.jsonc') then
```

Comment:
`vim.uv` only in nightly .   `vim.fn.filereadable(path ..'/.luarc.json')`  or `local uv = vim.version().minor >= 10 and vim.uv or vim.loop`

Reply:
`vim.loop` should be fine for now.